moveit_core/robot_state/include/moveit/robot_state/attached_body.h

```diff
@@ -110,6 +111,63 @@ class AttachedBody
     return attach_trans_;
   }
 
+  /** \brief Get the fixed transform to a subframe on this body (not a transform to a visual or collision shape).
+   * The frame_name needs to have the object's name prepended (e.g. "screwdriver/tip" returns true if the object's
+   * name is "screwdriver"). Returns an identity transform if frame is not found. 
```

Comment:
Documentation is not correct here: it throws instead of returning an identity transform or returning true.

Reply:
Fixed